src/dotnet/GeneratorAgent/src/Azure.Tools.GeneratorAgent/ProcessExecutionService.cs

```diff
@@ -194,7 +194,7 @@ private void LogExecutionResult(bool success, int exitCode, string command, stri
             else
             {
                 Logger.LogError("Command {Command} failed with exit code {ExitCode}. Error output: {ErrorOutput}",
-                    command, exitCode, error);
+                    command, exitCode, output);
```

Comment:
Curious - would it make sense to write both the error and the output here or is the output more informative?